sunpy_sphinx_theme/conf.py

```diff
@@ -1,22 +1,45 @@
 import os
 try:
-    from urllib.parse import urljoin
+    from urllib.parse import urljoin, urlparse
 except ImportError:
-    from urlparse import urljoin
-from . import get_html_theme_path
+    from urlparse import urljoin, urlparse
+import socket
+
+from jinja2 import Environment
+
+from sunpy_sphinx_theme import get_html_theme_path
 
 html_theme_path = get_html_theme_path()
 html_theme = "sunpy"
+html_static_path = [os.path.join(html_theme_path[0], html_theme, "static")]
 html_favicon = os.path.join(html_theme_path[0], html_theme, "static", "img", "favicon-32.ico")
 
+sunpy_website_url_base = socket.gethostname()
```

Comment:
interesting. I would want to check this with github pages to actually make sure it returns sunpy.org

Reply:
Yeah, I need to see how it works on deployment.